modules/weko-items-ui/weko_items_ui/views.py

```diff
@@ -1149,3 +1150,18 @@ def session_validate():
         "msg": _('Your session has timed out. Please login again.')
     }
     return jsonify(result)
+
+
+@blueprint_api.route('/check_public_status/<string:pid_value>', methods=['GET'])
```

Comment:
You are currently checking DOI, not record's status, `check_public_status` does not make sense.
Suggest: `check_doi_status` or `check_record_doi`